AK/StringBuilder.cpp

```diff
@@ -53,6 +53,13 @@ ErrorOr<void> StringBuilder::try_append(char ch)
     return {};
 }
 
+ErrorOr<void> StringBuilder::try_append_repeated(char ch, size_t n)
+{
+    for (size_t i = 0; i < n; ++i)
```

Comment:
Please at least ensure there's space before doing this, otherwise it could end up doing `n` reallocations.